arm-compute/2015-11-01-preview/swagger/containerService.json

```diff
@@ -0,0 +1,516 @@
+            "schema": {
+              "$ref": "#/definitions/ContainerService"
+            }
+          },
+          "201": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/ContainerService"
+            }
+          },
+		  "202": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/ContainerService"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true
+      },
+      "get": {
+        "tags": [
+          "ContainerService"
+        ],
+        "operationId": "ContainerService_Get",
+        "description": "The operation to get a container service.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "containerServiceName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the container service within the given subscription and resource group."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/ContainerService"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "ContainerService"
+        ],
+        "operationId": "ContainerService_Delete",
+        "description": "The operation to delete a container service.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "containerServiceName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the container service within the given subscription and resource group."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": ""
+          },
+          "204": {
+            "description": ""
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ContainerService/containerServices": {
+      "get": {
+        "tags": [
+          "ContainerService"
+        ],
+        "operationId": "ContainerService_List",
```

Comment:
ContainerService_ListByResourceGroup will be more clear.
It will be nice if the service can support an operation to list all the container services under a subscription.